test/net/sourceforge/kolmafia/session/QuestManagerTest.java

```diff
@@ -2220,9 +2227,8 @@ void talkingToGrandpaWithNoteAndYarnAdvancesQuest() {
           assertTrue(InventoryManager.hasItem(GRANDMAS_MAP));
 
           var requests = builder.client.getRequests();
-          assertThat(requests, hasSize(2));
+          assertThat(requests, hasSize(1));
           assertPostRequest(requests.get(0), "/monkeycastle.php", "action=grandpastory&topic=note");
-          assertPostRequest(requests.get(1), "/api.php", "what=status&for=KoLmafia");
```

Comment:
As I mentioned, since you are not expecting a second response, you can remove the ```builder.client.addResponse(200, ""); // api.php```

It's not harmful to leave it, so I'm approving, but if you remove it, don't wait for another approval.

Reply:
Removed.  I'm in no hurry...